pkg/resource/metadata_test.go

```diff
@@ -187,7 +187,7 @@ finalizers:
 	assert.True(t, md.Equal(in))
 }
 
-var ts, _ = time.Parse(time.RFC3339, "2021-06-23T19:22:29Z")
+var ts, _ = time.Parse(time.RFC3339, "2021-06-23T19:22:29Z") //nolint:errcheck
```

Comment:
Linter is probably right here - we do want to ensure that `err` is `nil` I suggest something like this
```suggestion
var ts = func() time.Time {
	res, err := time.Parse(time.RFC3339, "2021-06-23T19:22:29Z")
	if err != nil {
		panic(err)
	}

	return res
}()
```

Reply:
Okay, although I think if we can check things during assignments , we should do this. I usually just wrap the expression in `func must[T any](v T, err error) T`. Actually we should probably have `testutils` in `gen` so we could stop writing `must` for every package.